Running `sanity exec` results in Cannot find module '../../util/requireContext' error

**Describe the bug**

Running `sanity exec my/script.js` trigger the following error stack:

```
Error: Cannot find module '../../util/requireContext'
Require stack:
- /private/var/www/sanity/node_modules/@sanity/core/lib/actions/exec/requireContext.js
- internal/preload
    at Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._resolveFilename (/private/var/www/sanity/node_modules/@sanity/plugin-loader/loader.js:156:16)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/private/var/www/sanity/node_modules/@sanity/core/lib/actions/exec/requireContext.js:3:24)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Module._compile (/private/var/www/sanity/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Object.newLoader [as .js] (/private/var/www/sanity/node_modules/pirates/lib/index.js:104:7) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/private/var/www/sanity/node_modules/@sanity/core/lib/actions/exec/requireContext.js',
    'internal/preload'
  ]
}
```

By looking at the sanity core sources, the file util/requireContext.js is indeed [missing](https://github.com/sanity-io/sanity/tree/next/packages/%40sanity/core/src/util).

**Which versions of Sanity are you using?**

@sanity/cli                 2.21.10 (up to date)
@sanity/base                2.21.10 (up to date)
@sanity/components           2.14.0 (up to date)
@sanity/core                2.21.10 (up to date)
@sanity/dashboard           2.21.10 (up to date)
@sanity/default-layout      2.21.10 (up to date)
@sanity/default-login       2.21.10 (up to date)
@sanity/desk-tool           2.21.10 (up to date)
@sanity/language-filter     2.21.10 (up to date)
@sanity/production-preview   2.15.0 (up to date)
@sanity/vision              2.21.10 (up to date)

**What operating system are you using?**

Macos mojave

**Which versions of Node.js / npm are you running?**

8.1.2
v14.18.1

--- a/packages/@sanity/core/src/actions/exec/execScript.js
+++ b/packages/@sanity/core/src/actions/exec/execScript.js
@@ -32,12 +32,9 @@ module.exports = async function execScript(args, context) {
 
   const babel = require.resolve('./babel')
   const loader = require.resolve('./pluginLoader')
-  const requireContextPath = require.resolve('./requireContext')
   const browserEnvPath = require.resolve('./registerBrowserEnv')
   const configClientPath = require.resolve('./configClient')
-  const baseArgs = mockBrowserEnv
-    ? ['-r', browserEnvPath]
-    : ['-r', babel, '-r', loader, '-r', requireContextPath]
+  const baseArgs = mockBrowserEnv ? ['-r', browserEnvPath] : ['-r', babel, '-r', loader]
 
   const nodeArgs = baseArgs
     .concat(withUserToken ? ['-r', configClientPath] : [])

--- a/packages/@sanity/core/src/actions/exec/requireContext.js
+++ b/packages/@sanity/core/src/actions/exec/requireContext.js
@@ -1,3 +1,0 @@
-const requireContext = require('../../util/requireContext')
-
-requireContext.register()

